lib/classes/board.rb

```diff
@@ -0,0 +1,77 @@
+module OurTicTacToe
+	class Board
+		attr_reader :grid
+		def initialize(grid_board = {})
+			@grid = grid_board.fetch(:grid, default_board)
+		end
+
+		def get_cell(row, col)
+			grid[row][col]
+		end
+
+		def set_cell(row, col, value)
+			cell = get_cell(row, col)
+			cell.value.empty? ? cell.value = value : nil
+		end
+
+		def game_over
+			return :winner if winner?
+			return :draw if draw?
+
+			false
+		end
+
+		def formatted_grid
+			grid.each_with_index do |row, outer|
+				r = row.each_with_index.map do |col, inner|
+					if col.value.empty?
+						case outer
+						when 0
+							inner + 1
+						when 1
+							inner + 4
+						when 2
+							inner + 7
+						end
```

Comment:
If we use math, we can get rid of the `case` block. 
The three operations are very similar: `inner + number`
Which statement would allow us to get `1,4,7`? Use the elements you have: `outer`, `inner`

Reply:
We can use math that's how we originally got those numbers but we would still need conditionals.

Reply:
Thanks changed to `inner +  1 + (3 * outer)` and fixed in commit [58ce57a874c05da88f649f70797996c8c6e14e78](https://github.com/adriaanbd/tic-tac-toe/pull/7/commits/58ce57a874c05da88f649f70797996c8c6e14e78)